kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/form-templates-providers/js/kieserver-ui.js

```diff
@@ -340,15 +340,18 @@ function encodeImageFileAsURL(input) {
 		
 		var fileReader = new FileReader();
 
-		fileReader.onload = function(fileLoadedEvent) {
-			var local = fileLoadedEvent.target.result; // <--- data: base64
-			var srcData = local.replace(/^data:.*\/.*;base64,/, "");
-
-			fileData.get(input.id).data = srcData;
-			
-			console.log("Converted Base64 version is " + srcData);
+		fileReader.onload = function (fileLoadedEvent) {
+			if (fileReader.readyState == 2) {
+				var local = fileLoadedEvent.target.result; // <--- data: base64
+				var srcData = local.replace(/^data:.*\/.*;base64,/, "");
+
+				fileData.get(input.id).data = srcData;
+				fileReader.readAsDataURL(fileToLoad);
+				// console.log("Converted Base64 version is " + srcData);
```

Comment:
@bxf12315  Could you please remove this comment?

Reply:
@pefernan The code created by Maciej, if srcaData is very huge, the cost is very high, what do you think we remove it?

Reply:
@bxf12315 honestly I'd keep it as it is for now (or with small changes) since right now the idea is to post the docs content as a part of a json object. So we don't have much more options than reading the doc content here. 
I think the general assessment on forms is that they are not inteded to upload heavy documents, so I'd say that this is ok keep it like this.  If there's a need to attach big documents, I'd recommend other strategies than using a form.

Reply:
Back to the first message in this thread - please remove the commented line.